pekko-sample-persistence-dc-java/src/main/resources/application.conf

```diff
@@ -52,8 +52,8 @@ akka.persistence.cassandra {
 datastax-java-driver.advanced.reconnect-on-init = true
 
 
-# Akka Management config: https://developer.lightbend.com/docs/akka-management/current/index.html
-akka.management {
+# Apache Pekko Management config: https://developer.lightbend.com/docs/akka-management/current/index.html
```

Comment:
can you use a pekko url as opposed to a lightbend one?

Reply:
https://pekko.apache.org/docs/pekko-management/current/ is 404 not found

Reply:
I've updated to https://pekko.apache.org/docs/pekko-management/current/ since that's what we eventually anticipate here